presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -88,9 +88,10 @@ public IcebergNativeMetadata(
             IcebergResourceFactory resourceFactory,
             TypeManager typeManager,
             JsonCodec<CommitTaskData> commitTaskCodec,
-            CatalogType catalogType)
+            CatalogType catalogType,
+            String prestoVersion)
```

Comment:
Yes, because createView implementation is there in IcebergAbstractMetadata. I would prefer to keep it this way, so that we can add support for view creation in nessie catalog in future. IcebergAbstractMetadata is the base class